index.js

```diff
@@ -1,6 +1,8 @@
 import AddBookClass from './modules/addBooks.js';
 import DisplayBooksClass from './modules/displayBooks.js';
 import RemoveBook from './modules/removeBook.js';
+// eslint-disable-next-line import/no-unresolved, import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
```

Comment:
- [ ] I can see you imported Luxon from node module which is also working but there is something you missed out on this requirement kindly consider adding this package as  .js file to the project (in full or minified version) beause we have to  make sure that the Luxon library is used not just importing it from node modules and also consider importing it this way `import { DateTime } from "luxon"`

Reply:
Thank you @divinecharlotte , I had done it this way previously however the ESLint did not pass with the luxon.js full. Kindly advise any workaround this issue since I am not supposed to modify the ESLint config file.